src/Analytics/Adapter/HubSpot.php

```diff
@@ -57,30 +57,19 @@ public function getName(): string
     public function contactExists(string $email): bool|int
     {
         try {
-            $result = $this->call('POST', '/crm/v3/objects/contacts/search', [
```

Comment:
What changed here?

Reply:
I switched the API from the search one to the get contact one using the `idProperty` param, this should handle contacts that have secondary emails where as search doesn't causing an edge condition